client/checkout/style.scss

```diff
@@ -9,25 +9,46 @@
 	border: none !important;
 	// some themes might override the color on all `button`s - whose selector has higher specificity and our class selector.
 	background-color: transparent !important;
-	padding: 3px;
+	font-weight: normal;
+	display: inline-flex;
+	cursor: pointer;
+	color: inherit;
+	font-size: initial;
+	padding: 2px 1px;
+	align-items: center;
+
+	span {
+		margin-right: 4px;
+	}
 
 	i {
-		display: inline-block;
+		order: 1;
```

Comment:
I tried to do that in the `get_testing_instructions()` pattern, but the replacement done by `WC_Payments_Utils::esc_interpolated_html()` never really worked for me :( 
it looks like `WC_Payments_Utils::esc_interpolated_html` attempts to close any opened HTML tag. I couldn't figure out how to open an `<i>` after the `<button>`'s open tag, but before the `<button>`'s tag got closed :suboptimal:

Let me know if you have a better solution! And sorry I missed clarifying this in a comment, it was quite tricky to get the markup right :D

Reply:
Ah, ok. Yeah I see how it's a pain. Would this work? 

```
// class-cc-payment-method.php - line 72
public function get_testing_instructions() {
	return __( '<strong>Test mode:</strong> use test card <button><span>4242 4242 4242 4242</span><i></i></button> or refer to our <a>testing guide</a>.', 'woocommerce-payments' );
}
```

```
// class-wc-payments-checkout.php - line 326
$settings[ $payment_method_id ]['testingInstructions'] = WC_Payments_Utils::esc_interpolated_html(
	/* translators: link to Stripe testing page */
	$payment_method->get_testing_instructions(),
	[
		'a'      => '<a href="https://woocommerce.com/document/woopayments/testing-and-troubleshooting/testing/#test-cards" target="_blank">',
		'strong' => '<strong>',
		'button' => '<button type="button" class="js-woopayments-copy-test-number" aria-label="' . esc_attr( __( 'Click to copy the test number to clipboard', 'woocommerce-payments' ) ) . '" title="' . esc_attr( __( 'Copy to clipboard', 'woocommerce-payments' ) ) . '">',
		'i'      => '<i>',
		'span'   => '<span>',
	]
);
```

Reply:
Yes, that works - however, I wanted to ensure the markup in the translatable string was kept to a minimum to simplify the translators' job. It would be easy to use the wrong markup and not have the correct result in the interpolation. By just having the `<button>` tag, we're limiting the risk of using the wrong markup in case the translators don't understand how the HTML tags don't fit into the string.

In short, I was trying to simplify the HTML in the string - but I can be persuaded :D what do you think?

Reply:
That's fair and I think more important than the order of the elements. Thanks for explaining!